react-app-rewired/index.ts

```diff
@@ -350,6 +351,17 @@ const reactAppRewireConfig = {
       }),
     )
 
+    _.merge(config, {
+      plugins: [
+        ...(config.plugins ?? []),
+        // Webpack is unable to resolve imports within the @cowprotocol/app-data package due to
+        // expressions in the imports, resulting in a compiler warning. As a result, Webpack
+        // bundles all imports from this package. This plugin silences the warning.
+        // https://webpack.js.org/guides/dependency-management/#require-with-expression
+        new ContextReplacementPlugin(/@cowprotocol\/app-data/),
+      ],
+    })
```

Comment:
Tried bumping cowswap version to latest - didnt help but broke tests, so reverted to current version.

Reply:
Weird that we didn't have this warning before, wait, did we have it? What made it out loud?

Reply:
Its due to the different compiler options of packages vs web, where webpack is unable to interpret the import as an expression unlike the compiled output of web which uses different configuration